checador/spec/helpers/companies_helper_spec.rb

```diff
@@ -1,5 +1,15 @@
 require 'rails_helper'
 
+# Specs in this file have access to a helper object that includes
+# the CompaniesHelper. For example:
+#
+# describe CompaniesHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
```

Comment:
If you do not occupy this code, delete it.

Reply:
Thanks, the changes were made, could you check